staging/src/k8s.io/dynamic-resource-allocation/controller/controller_test.go

```diff
@@ -170,7 +179,11 @@ func TestController(t *testing.T) {
 			claim:         withFinalizer(withDeletionTimestamp(claim), ourFinalizer),
 			driver:        m.expectDeallocate(map[string]error{claimName: errors.New("fake error")}),
 			expectedClaim: withFinalizer(withDeletionTimestamp(claim), ourFinalizer),
-			expectedError: "stop allocation: fake error",
```

Comment:
I like that the unit test depends a lot less on implementation details. Instead of checking what goes on while processing one item, we check the outcome.

/lgtm
/approve

Reply:
learned!